src/main/java/net/greatstart/dao/hibernate/EventDaoImpl.java

```diff
@@ -3,6 +3,7 @@
 import net.greatstart.dao.EventDao;
 import net.greatstart.model.Event;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.transaction.annotation.Transactional'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)